packages/x-charts/README.md

```diff
@@ -1,5 +1,33 @@
 # @mui/x-charts
 
-Package under development.
+This package is the community edition of the charts components.
+It's part of MUI X, an open-core extension of MUI, with advanced components.
 
-The alpha phase will start in the coming months, stay tuned!
+## Installation
+
+Install the package in your project directory with:
+
+```sh
+npm install @mui/x-charts
+```
+
+or
+
+```sh
+yarn add @mui/x-charts
+```
+
+This component has the following peer dependencies that you will need to install as well.
+
+```json
+"peerDependencies": {
+    "@mui/material": "^5.4.1",
```

Comment:
I think that a peer dependency will be a major blocker for the broader adoption of the chart library. It's OK in the near future, but I think that we need to have a plan for how to make it optional.

Reply:
That is a great observation! 👌 
It looks like we mainly depend on the styling system, which can easily be replaced with imports from the `@mui/system`.
But there are a few usages of actual `material` imports like `Popper`, `Paper`, and `NoSsr`, which shouldn't be too difficult to replace. 
cc @alexfauquette